src/api/partner-campaign/content-types/partner-campaign/lifecycles.ts

```diff
@@ -0,0 +1,35 @@
+import type { Event } from '@strapi/database/dist/lifecycles'
+import { Marked, Renderer } from 'marked'
+import slugify from 'slugify'
+
+const renderer = new Renderer()
+const marked = new Marked({ renderer })
+
+const computedFieldsHook = async (event: Event) => {
+  const partnerCampaign = event.params.data
+
+  if (!partnerCampaign) {
+    return
+  }
+
+  if (!partnerCampaign.slug) {
+    partnerCampaign.slug = slugify(String(partnerCampaign.slug), {
+      lower: true,
+    })
+  }
+
+  if (partnerCampaign.content) {
+    partnerCampaign.htmlContent = await marked.parse(partnerCampaign.content)
```

Comment:
Open question. Do we need id and anchors ?

Reply:
Pas sur la landing SNCF et pas sûr que le besoin émerge on peut sans doute virer

Reply:
j'ai des saut de ligne inopinés en utilisant le content, je continue d'utiliser htmlContent